### PR TITLE
[CDU] Fix airways showing as undefined

### LINKED ISSUE #2585: [BUG] 'UNDEFINED' appears against all fpln wps
**Mod Version**
2020-12-21T10:11:43+00:00

**Describe the bug**
Going through MCDU flow after INIT and AOC INIT (import from Simbrief etc) on checking FPLN page 'UNDEFINED' appears above every waypoint

**To Reproduce**
1. Prep Simbrief flightplan
2. Use Simbrief integration w/flow

**Expected behavior**
Clear flightplan with any airways or routing

**Actual behavior**
'UNDEFINED' above each waypoint in MCDU
EDIT: I note there is no CRZ Flight Level in PROG Page

**References**
Screenshot 2020-12-21 150541

**Additional context**

Was this working before/when did the issue start occurring?
Was working fine this morning before upgrade to this version

Is this a problem in the vanilla unmodded game?
Don't know

### PR DESCRIPTION
**Pull request by pareil6**
Fixes #2585.

## Summary of Changes
Fixes airways field in FPLN page showing as 'undefined' when between two waypoints without a connecting airway. Issue occurred where any two waypoints both had an undefined outbound/inbound airway, which would incorrectly pass an equality check.

## Screenshots (if necessary)
Before:
image

After:
image

## References
N/A

## Additional context
N/A

Discord username (if different from GitHub): pareil6#0990

## Testing instructions
Check 'undefined' never shows up as an airway name - try SIDs/STARs, random points within a flight plan, last waypoint before STAR/first after SID/etc. Also, could be worth checking nothing strange happens once in flight as you pass waypoints.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by MisterChocker — APPROVED** [review]
LGTM

**Comment by MisterChocker** [comment]
Please add changelog

**Comment by pareil6** [comment]
> Please add changelog

Dammit, every time... Done 😛

**Comment by MisterChocker** [comment]
Tested and works.

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 1. [CDU] Fix incorrect block time display on INIT/REVIEW - @pareil6 - (pareil6)
 1. [ECAM] Improved PACKS indication - @MisterChocker (Leon)
 1. [CDU] Improved visuals of CDU Pages - @MisterChocker (Leon)
+1. [CDU] Fix FPLN airways showing as undefined
```
Comment (MisterChocker) [inline .github/CHANGELOG.md:12]:
```suggestion
1. [CDU] Fixed FPLN airways showing as undefined - @pareil6 - (pareil6)
```

**Review by MisterChocker — CHANGES_REQUESTED** [review]

**Review by MisterChocker — APPROVED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js [hunk 2]
```diff
@@ -256,7 +256,7 @@ class CDUFlightPlanPage {
                         let airwayName = "";
                         if (prevWaypoint && waypoint) {
                             let airway = undefined;
-                            if (prevWaypoint.infos.airwayOut === waypoint.infos.airwayIn) {
+                            if (prevWaypoint.infos.airwayOut && (prevWaypoint.infos.airwayOut === waypoint.infos.airwayIn)) {
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js:259]:
```suggestion
                            if (prevWaypoint.infos.airwayOut && prevWaypoint.infos.airwayOut === waypoint.infos.airwayIn) {
```

Comment (MisterChocker) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js:259]: missed these parentheses before, they are not needed btw

**Review by MisterChocker — CHANGES_REQUESTED** [review]

**Review by MisterChocker — APPROVED** [review]

**Comment by NickFarin** [comment]
Category: QA Tester
Name: Nick F[Z+8]#5327
Date of testing: 22/12/2020
Version of the sim: 1.11.7.0
PR Tested: #2589 
Object of test: CDU airways showing as undefined

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Spawned at KLAS
2. Made a flight plan on SimBreif
3. Pressed Init request
4. Checked if there are any undefined airways

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Test was nominal.

**Pull request merged**